icicle/src/polynomials/cuda_backend/kernels.cuh

```diff
@@ -0,0 +1,104 @@
+#pragma once
+#include "stdint.h"
+#include "../../../src/vec_ops/vec_ops.cu" // TODO Yuval: avoid this
```

Comment:
Can we work around it?

Reply:
Yes I can expose them like the add/sub/mul by decalring in vec_ops.cuh and intantiate in vec_ops/extern.cu. I was just being a little lazy.
Alternatively I can move them back to the cuda_backend/kernels.cuh like it was.
It is mainly a question of what we want to expose in vec_ops I think. What we don't I will have in the cuda_backend//kernels.cuh and what we do I can expose like add/sub/mul.

So I will do that I think.